### PR TITLE
Add ICICI Bank

### LINKED ISSUE #6020: Add ICICI Bank
**Brand Name:** ICICI Bank
**Website:** https://www.icicibank.com/
**Similarweb rank:** [892](https://www.similarweb.com/website/icicibank.com/#overview)

**Official resources for icon and color:**
Wikipedia. Primary color used from the logo is `#AE282E`.

### PR DESCRIPTION
**Pull request by wwwescape**
icicibank

**Issue:** Closes #6020
**Similarweb rank:** [892](https://www.similarweb.com/website/icicibank.com/#overview)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
 - I used the SVG from Wikipedia
 - I used Inkscape to remove the text, scale and adjust
 - I used https://jakearchibald.github.io/svgomg/ to optimize

### CODE REVIEW

**Comment by PeterShaggyNoble** [comment]
Thanks for the contribution, @wwwescape 👍🏻

Found an official vector version available in the PDFs on [this page](https://www.icicibank.com/aboutus/annual.page) that we should base our version off - would you mind updating based on one of them? I also wonder if we shouldn't try a different treatment to try to provide some separation between the coloured sections? Normally we'd go with a cut-out but I gave that a rough try myself and it doesn't look great.

For the colour, should we go with the orange instead of the red? It was the colour that immediately jumped out at me when I visited their website and looks to be used a little bit more. Being, presumably, more familiar with the brand than I, I'll defer to whichever you choose.

**Comment by wwwescape** [comment]
Sure @PeterShaggyNoble. I'll take another look at it over the weekend.

**Comment by wwwescape** [comment]
@PeterShaggyNoble I tried my best with the vectors from the PDF. Could you check if the updated SVG is good enough?

```
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ICICI Bank</title><path d="M17.594 0c-3.09-.014-6.982 1.82-10.419 5.23-5.495 5.453-7.84 12.934-5.214 16.69.912 1.268 2.275 1.946 3.89 2.08l-.13-.016c1.09.04 1.982-1.174 2.71-2.79.728-1.657 1.13-3.193 1.495-4.65.526-2.302.566-3.92.283-4.283-.444-.568-1.414-.444-2.549.202-.565.323-1.293.12-.442-.93.89-1.05 4.324-3.557 5.536-3.962 1.376-.363 2.912.163 2.387 1.658-.366 1.09-5.134 13.54-1.576 12.248l-.152.11a22.112 22.112 0 003.424-2.817c5.498-5.457 7.844-12.976 5.175-16.696a5.03 5.03 0 00-.31-.389C20.7.556 19.264.007 17.593 0zm-.931 1.838c.319.013.59.116.78.318.203.202.324.445.324.768v.121c-.04.687-.445 1.536-1.172 2.263-1.091 1.05-2.545 1.455-3.153.85-.687-.646-.281-2.062.81-3.194.778-.75 1.708-1.156 2.41-1.126z"/></svg>
```

I don't think we need to provide some separation between the colored sections. They show it the same way for their mobile notifications.

image

For the color, the strange thing is when viewing the site publicly it's all orange but when I log in, everything is in maroon. Probably why I leaned towards maroon. Also, their branch decor and [staff uniforms](https://qr.ae/pGH0O6) have maroon as their stand-out color.

Let me know which one you want me to use.

Color `#F06321` (from Logo):
icicibank

Color `#AE282E` (from Logo):
icicibank (alternate color)

Color `#` (from website):
icicibank (website color)

File: _data/simple-icons.json [hunk 1]
```diff
@@ -6160,6 +6160,11 @@
             "hex": "CC092F",
             "source": "https://www.iceland.co.uk"
         },
+        {
+            "title": "ICICI Bank",
+            "hex": "AE282E",
+            "source": "https://commons.wikimedia.org/wiki/File:ICICI_Bank_Logo.svg"
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:6166]: Can we use this instead?
```suggestion
            "source": "https://www.icicibank.com/ms/aboutus/annual-reports/2022-23/icici/assets/images/home-page/logo.svg"
```

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (PratyushJayachandran) [reply _data/simple-icons.json:6166]: @adamrusted Please take a look. We can use source SVG from the official website itself

Comment (adamrusted) [reply _data/simple-icons.json:6166]: @PratyushJayachandran - if it's on the homepage, we should instead use the homepage URL.

Comment (PratyushJayachandran) [reply _data/simple-icons.json:6166]: It's actually not available as svg on the homepage that's why I put this link.

File: icons/icicibank.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ICICI Bank</title><path d="M17.594 0c-3.09-.014-6.982 1.82-10.419 5.23-5.495 5.453-7.84 12.934-5.214 16.69.912 1.268 2.275 1.946 3.89 2.08l-.13-.016c1.09.04 1.982-1.174 2.71-2.79.728-1.657 1.13-3.193 1.495-4.65.526-2.302.566-3.92.283-4.283-.444-.568-1.414-.444-2.549.202-.565.323-1.293.12-.442-.93.89-1.05 4.324-3.557 5.536-3.962 1.376-.363 2.912.163 2.387 1.658-.366 1.09-5.134 13.54-1.576 12.248l-.152.11a22.112 22.112 0 003.424-2.817c5.498-5.457 7.844-12.976 5.175-16.696a5.03 5.03 0 00-.31-.389C20.7.556 19.264.007 17.593 0zm-.931 1.838c.319.013.59.116.78.318.203.202.324.445.324.768v.121c-.04.687-.445 1.536-1.172 2.263-1.091 1.05-2.545 1.455-3.153.85-.687-.646-.281-2.062.81-3.194.778-.75 1.708-1.156 2.41-1.126z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/icicibank.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>ICICI Bank</title><path d="M21.9258 2.0961C19.279-1.6476 12.698-.2426 7.2138 5.2416c-5.484 5.475-7.7865 12.9625-5.1397 16.7062.8728 1.2386 2.1837 1.902 3.7386 2.0522 1.0516.0078 1.9129-1.1846 2.6158-2.7774.7252-1.6678 1.1694-3.218 1.5138-4.6592.5077-2.2934.544-3.934.29-4.2786-.435-.5801-1.4321-.435-2.5561.2176-.544.2991-1.26.0997-.408-.9336.8612-1.0425 4.2605-3.5625 5.4933-3.9523 1.3415-.3898 2.8734.136 2.3568 1.6226-.3706 1.0847-5.0473 13.486-1.596 12.2719 1.1049-.747 2.205-1.6497 3.2639-2.7086 5.4841-5.475 7.7865-12.9625 5.1396-16.7063zm-5.3662 3.209c-1.0969 1.0968-2.52 1.4865-3.1364.852-.6617-.6345-.272-2.0577.8249-3.1726 1.1058-1.115 2.529-1.4594 3.1454-.834.6345.6436.2448 2.0487-.834 3.1545z"/></svg>
```

**Comment by PratyushJayachandran** [comment]
> @PratyushJayachandran - I'm seeing [some differences](https://github.com/simple-icons/simple-icons/assets/1830867/1e33d1ad-b362-4743-9e8c-375e5bba22d6) between source (orange+black) and your contrib (red). Potentially just a precision issue?

Redid from source. Please check again.

**Review by adamrusted — CHANGES_REQUESTED** [review]
@PratyushJayachandran - weirdly, still seeing differences.
image
Unsure what else to suggest to try and resolve this...

**Comment by PratyushJayachandran** [comment]
> @PratyushJayachandran - weirdly, still seeing differences. image Unsure what else to suggest to try and resolve this...

image
Looks correct on my system 💀

File: _data/simple-icons.json (hunk 1 above)
Comment (PeterShaggyNoble) [reply _data/simple-icons.json:6166]: @PratyushJayachandran, can you update the URL to the page you found the SVG on?

Comment (PratyushJayachandran) [reply _data/simple-icons.json]: > @PratyushJayachandran, can you update the URL to the page you found the SVG on?

Done

**Comment by adamrusted** [comment]
@PratyushJayachandran - you're not going to believe this. The difference was because I hadn't merged the main `i` shape into the oval. It extends the shape ever so slightly...

**Pull request merged**